### ISSUE TITLE
[Feature]: Add option for evaluation metric

### ISSUE THREAD

**Issue by fritshermans** [dynamiq-ai/pragmatiq#22]
### Problem

Currently, AUCROC is used as validation metric for finetuning. As some of the Pragma applications involve imbalanced classification (e.g. AML, Fraud), it would be good if the user could choose to use sklearn's average precision (=AUC of precision recall curve) as validation metric (`sklearn.metrics.average_precision_score`).

### Proposed solution

The solution would be to add the option `validation_metric` to the `api.finetune` function that defaults to `aucroc` and that could also take the option `ap` for average precision (or `aucpr`)

### Area

Training or finetuning
